Sources/NIO/ByteBuffer-core.swift

```diff
@@ -49,6 +49,13 @@ extension _ByteBufferSlice {
     }
 }
 
+extension _ByteBufferSlice: CustomStringConvertible {
+    @usableFromInline
+    var description: String {
+        return "_ByteBufferSlice { \(self.lowerBound)..<\(self.upperBound) }"
```

Comment:
Should this print the parent buffer?

Reply:
probably yes, @JovanMilenkovic happy adding that? Otherwise we can also do this as a followup

Reply:
I’ll see if I could check it out later tonight, but if I remember correctly `ByteBuffer` has it’s own description that also contains the slice. Wouldn’t that double print the whole buffer and cause a confusing print or am I missing something?

Reply:
Also potentially infinite if it returns the full description itself...

Reply:
@jovanmilenkovic you’re right. I thought it was ByteBufferView but this is the slice so your code is fine as is. Sorry

Reply:
Oof, so did I. The price you pay for reviewing on a phone.